docs/ide.md

```diff
@@ -0,0 +1,71 @@
+# IDE Integration
```

Comment:
I would remove this, and add only:
```
# IDE Integration

In your `phpinsights.php` file, add the conf...

## Supported IDE 
...
## Unsupported IDE
...
```

Reply:
Do you mean without the **prerequisite & troubleshouting** part ? 

Should i display them as [tip or warning](https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers) instead ?